packages/mlpost-lablgtk/mlpost-lablgtk.0.9/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "filliatr@lri.fr"
+authors: [
+  "Romain Bardou"
+  "Francois Bobot"
+  "Jean-Christophe Filliâtre"
+  "Johannes Kanig"
+  "Stephane Lescuyer"
+]
+homepage: "https://github.com/backtracking/mlpost"
+dev-repo: "git+https://github.com/backtracking/mlpost.git"
+bug-reports: "https://github.com/backtracking/mlpost/issues"
+doc: "https://backtracking.github.io/mlpost"
+synopsis: "Library for adding mlpost graphics in lablgtk"
+description: "Library for adding mlpost graphics in lablgtk"
+license: "LGPL-2.1"
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+depends: [
+  "ocaml" { >= "4.07.0" }
+  "bitstring" { >= "3.0.0" }
+  "dune" { >= "2.7.0" }
+  "mlpost"
```

Comment:
Are you planning on releasing `mlpost-lablgtk` alongside `mlpost`? If yes, I would highly suggest adding `{= version}` here.

Reply:
Thanks. Makes sense. I'll add this constraint.